tick.q

```diff
@@ -27,7 +27,7 @@ if[not system"p";system"p 5010"]
 \l tick/u.q
 \d .u
 ld:{if[not type key L::`$(-10_string L),string x;.[L;();:;()]];i::j::-11!(-2;L);if[0<=type i;-2 (string L)," is a corrupt log. Truncate to length ",(string last i)," and restart";exit 1];hopen L};
-tick:{init[];if[not min(`time`sym~2#key flip value@)each t;'`timesym];@[;`sym;`g#]each t;d::.z.D;if[l::count y;L::`$":",y,"/",x,10#".";l::ld d]};
+tick:{[x:{init[];$[not min(`time`sym~2#key flip value@)each t;'`timesym;x]};y]@[;`sym;`g#]each t;d::.z.D;if[l::count y;(L;l):(`$":",y,"/",x,10#".";ld d)]};
```

Comment:
`L, l` are globlal variables, so you should put:
```
(L;l)::(`$":",y,"/",x,10#".";ld d)]};
```

Reply:
True, thanks!